pkgs/development/tools/stem/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchurl
+, python3
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "stem";
+  version = "1.8.2";
+
+  src = fetchurl {
+    url = "https://gitlab.torproject.org/tpo/network-health/stem/-/archive/1.8.2/stem-1.8.2.tar.gz";
+    hash = "sha256-RuHyrFZLDdjw2IwQjX9F5cFqQ2vYuG0W/tlcMnskd1k=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [ setuptools ];
+
+  doCheck = false;
```

Comment:
There are tests, please either solve problems around it or leave a comment near this line to explain why.